src/modules/job-manager/job.h

```diff
@@ -37,6 +38,8 @@ struct job {
 
     json_t *annotations;
 
+    struct grudgeset *dependencies;
+
```

Comment:
In my fastrun branch, I added an aux hash to `struct job`.   Not a big deal since the grudgeset is created on demand, but if you wanted pull in eb543fc7b6221e8b559f655e8a540a9d6446b7c1 and put the grudge set in the aux hash, then even the cost of the pointer would be saved when there are no dependencies for a job (assuming the cost of the aux hash pointer is amortized over other features as well)

Reply:
ooh nice! 🤑